packages/react-composites/src/composites/MeetingComposite/state/MeetingAdapterState.ts

```
@@ -59,9 +24,8 @@ export interface MeetingAdapterUiState {
  */
 export interface MeetingAdapterClientState extends Pick<CallAdapterClientState, 'devices'> {
   userId: CommunicationIdentifier;
-  displayName: string;
-  latestErrors: MeetingErrors;
```

Feel free to ignore errors for now. We can add that back in later.